creator-node/src/fileManager.js

```diff
@@ -95,18 +95,35 @@ async function saveFileToIPFSFromFS (req, srcPath) {
  *                  eg original.jpg or 150x150.jpg
  */
 async function saveFileForMultihashToFS (req, multihash, expectedStoragePath, gatewaysToTry, fileNameForImage = null) {
+  // stores all the stages of this function along with associated information relevant to that step
+  // in the try catch below, if any of the nested try/catches throw, it will be caught by the top level try/catch
+  // so we only need to print it once in the global catch or after everthing finishes except for any return statements
```

Comment:
everything

Reply:
🙏

Reply:
hahah i meant there's a typo in comment above in the word "everything"